### PR TITLE
Never settle

### PR DESCRIPTION
**Pull request by dmcrodrigues**
Include the new `Dev Experience` squad.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,13 +65,14 @@ By definition, members work on their respective squad, although they are free to
...
+| Dev Experience                | Ilya, Olivier, David                        |    3/3       | |
 | Early Feedback                | Danilo                                      |    1/1       | [#early_feedback](https://babylonhealth.slack.com/archives/CLGA76GG7) |
 | Enrolment and Integrity       | Diego                                       |    1/1       | [#ei-squad](https://babylonhealth.slack.com/archives/CGR4D5NKX) |
 | Healthcheck                   | Ben, Julien                                 |    2/3       | [#healthcheck_tribe](https://babylonhealth.slack.com/archives/C7995CX3R) |
 | Monitor                       | Anders, Joshua, Nicola                      |    3/3       | [#patient-metrics-team](https://babylonhealth.slack.com/archives/CE37S5W9Z) |
-| Native Apps                   | Giorgos, Jason, Martin, Inami, Emese, David |    6/6       | [#nativeapps_public](https://babylonhealth.slack.com/archives/CE5P8LRNH) |
+| Native Apps                   | Giorgos, Jason, Martin, Inami, Emese        |    6/6       | [#nativeapps_public](https://babylonhealth.slack.com/archives/CE5P8LRNH) |
```
Comment (karabatov) [inline README.md]: What about `6/6`?

Comment (dmcrodrigues) [reply README.md]: Damn

**Review by j531 — APPROVED** [review]
Pending Yuri's change.

BTW, what's the remit of the Dev Experience squad?

**Review by chitrakotwani — APPROVED** [review]

**Review by karabatov — APPROVED** [review]
We really need 3 people for this now? My, my :)

**Comment by dmcrodrigues** [comment]
@j531 Dev Experience will be mostly focused in tooling, infrastructure and architecture.

**Pull request merged**